client/lib/lib.go

```diff
@@ -1274,13 +1274,11 @@ func containsUnescaped(query string, token string) bool {
 }
 
 func stripBackslash(query string) string {
-	runeQuery := []rune(query)
 	var newQuery []rune
-	for i := 0; i < len(runeQuery); i++ {
-		if runeQuery[i] == '\\' && i+1 < len(runeQuery) {
-			i++
+	for _, char := range query {
+		if char != '\\' {
+			newQuery = append(newQuery, char)
 		}
-		newQuery = append(newQuery, runeQuery[i])
```

Comment:
Hm, I am looking at this and I think there are possible a couple of issues with the simplification?
- I think the simplified version will simply strip all `\` characters, so you couldn't use `\\` to pass an actual backslash to the resulting string?
- As far as I understand a string and a rune array are different beasts? This would mean that as we loop through a string we get a byte each iteration as opposed to looping through a rune array we get a rune (i.e. a full UTF-8 character) each iteration?

Reply:
Ah thank you! You are totally right and I missed that when making tweaks. Fixed in ff24b66